components/mpas-albany-landice/src/mode_forward/mpas_li_advection.F

```diff
@@ -1012,6 +1036,7 @@ subroutine apply_mass_balance(&
                !TODO - If remaining sfcAblat > 0, then keep track of it to conserve energy (?)
             endif
 
+            bareIceAblationApplied(iCell) = 0.0_RKIND ! set bare ice ablation applied to zero where there is no ice
```

Comment:
See previous comment - this line should be removed after bareIceAblationApplied comes from sfcMassBalApplied, as that will catch this situation too.

Reply:
It seems like 'bareIceAblationApplied' should then be specified only after line 1041 (and not above at line 972). Because otherwise, that field will not include any adjustments made to 'sfcMassBalApplied' in lines 1034-1037.